src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/optimizer/utils/VarSpecificCardinalitiesCache.java

```diff
@@ -0,0 +1,35 @@
+package se.liu.ida.hefquin.engine.queryproc.impl.optimizer.utils;
+
+import org.apache.jena.sparql.core.Var;
+import se.liu.ida.hefquin.engine.queryplan.PhysicalPlan;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class VarSpecificCardinalitiesCache {
```

Comment:
This is the latest version of this class, right?

Reply:
Yes